src/main/java/org/myrobotlab/codec/CodecUtils.java

```diff
@@ -750,6 +751,18 @@ public static ServiceConfig readServiceConfig(String filename) throws IOExceptio
     return (ServiceConfig)yaml.load(data);
   }
 
+  public static void setField(Object o, String field, Object value) {

```

Comment:
Seems like you'll always pass a Service class in here instead of object... what's the rational for the fancy  reflection going on below?

Reply:
utilities should be generalized, and not dictate the way they are used